src/tools/tests/golden.arcs

```diff
@@ -16,3 +16,5 @@ particle Gold
   alias: writes {val: Text}
 
   collection: reads [{num: Number}]
+
+  foo: writes Foo {txt: Text}
```

Comment:
Could you achieve the same by modifying some existing connections by adding a schema name?

Golden should no longer act as our unit tests, but as a test of how all the pieces get integrate into a single file, as such I think we should minimize the number of handle connections on this particle.

Reply:
Done!